hollow-explorer-ui/src/main/resources/browse-selected-type-top.vm

```diff
@@ -37,7 +37,7 @@ NOTE: Results are filtered by query <span style="font-family: monospace;">$esc.h
 				<a href="$basePath/type?type=$esc.url($type)&key=$esc.url($key)&ordinal=$ordinal&display=$display&page=$nextPage&pageSize=$pageSize">NEXT</a>
 			#end
 		</td>
-		#if($ordinal)
+		#if($ordinal != $null)
```

Comment:
Have you verified if this null check works? I think checking against $null might cause some issues. 
Existing checks should suffice. The value null is tested as the boolean value false by Velocity. You could either add this - 
#if (!$variable)
 ... do stuff (nothing) here if the variable is null
#end
#if( $variable == true )
	...Variable is NOT null
#end
Ref - https://itecnote.com/tecnote/java-null-check-in-velocity/ 
http://oliviertech.com/java/velocity/how-to-check-if-a-value-is-null-with-velocity/

Reply:
Yes this is tested and its the most direct way to check null, relies on the variable $null not being assigned to anything. The other ways as you noted are more roundabout. Unfortunately simply doing `#if ($ordinal)` doesnt seem to distinguish between null and 0 value when ordinal is an int.